packages/nodejs.vm/nodejs.vm.nuspec

```
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>nodejs.vm</id>
-    <version>0.0.0.20250219</version>
+    <version>0.0.0.20250730</version>
     <authors>Node.js Foundation</authors>
     <description>Metapackage for Node.js to ensure all packages use the same Node.js version.</description>
     <dependencies>
       <dependency id="common.vm"  version="0.0.0.20250206" />
-      <dependency id="nodejs" version="[20.7.0, 20.8.0)" />
+      <dependency id="nodejs" version="[21.0.0]" />
```

you need to use the `[,)` version format as our bot updates exact versions weekly: https://github.com/mandiant/VM-Packages/blob/main/scripts/utils/update_package.py#L262

```suggestion
      <dependency id="nodejs" version="[21.0.0, 21.1.0)" />
```

---

Done.